admin/base/settings/defaults.py

```diff
@@ -23,9 +25,30 @@
     '.osf.io'
 ]
 
+AUTH_PASSWORD_VALIDATORS = [
+    {
+        'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',
+    },
+    {
+        'NAME': 'django.contrib.auth.password_validation.MinimumLengthValidator',
+        'OPTIONS': {
+            'min_length': 5,
```

Comment:
@zkraime Okay, so then we should either:

 1. drop the "Password length restriction" requirement from the [purpose](https://github.com/CenterForOpenScience/osf.io/pull/4992#issuecomment-196511342) of this PR
 1. meet the "Password length restriction" requirement another way
 1. upgrade to Django 1.9

Since this PR is so mondo already, I recommend (1). :-)